rascal-vscode-extension/package.json

```diff
@@ -20,9 +20,11 @@
     "url": "https://github.com/usethesource/rascal-language-servers"
   },
   "dependencies": {
+    "mac-ca": "^1.0.6",
```

Comment:
dependency added but not used?

Reply:
it's imported, that's enough.

see the comment:

```
import * as macca from 'mac-ca'; // just importing is is enough
```